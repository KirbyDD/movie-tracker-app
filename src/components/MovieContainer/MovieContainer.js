import React from 'react';
import GenreContainer from '../GenreContainer/GenreContainer';
import './MovieContainer.css';

const MovieContainer = () => {
  return (
    <main className='movie-container'>
      <GenreContainer />
      <GenreContainer />
      <GenreContainer />
      <GenreContainer />
      <GenreContainer />
    </main>
  )
}

export default MovieContainer;