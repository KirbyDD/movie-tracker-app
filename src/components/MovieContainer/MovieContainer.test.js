import React from 'react';
import {shallow} from 'enzyme';
import MovieContainer from './MovieContainer';

describe('Movie Container', () => {
  let wrapper;
  beforeEach(function() {
    wrapper = shallow(
      <MovieContainer/>
    )
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
