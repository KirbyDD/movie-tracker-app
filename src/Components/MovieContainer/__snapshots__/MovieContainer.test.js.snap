// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Container should match snapshot 1`] = `
<main
  className="movie-container"
>
  <Connect(GenreContainer)
    title="ACTION"
  />
</main>
`;
